packages/pn-personagiuridica-webapp/src/pages/Deleghe.page.tsx

```diff
@@ -52,6 +52,11 @@ const Deleghe = () => {
     };
   }, []);
 
+  useEffect(() => {
```

Comment:
Using the useEffect is a good intuition.
It can be change to make it more specific

`
useEffect(() => {
    const tabToBeSelected = pathname === routes.DELEGATI ? 1 : 0;
    if (value !== tabToBeSelected) {
      setValue(tabToBeSelected);
    }
  }, [pathname]);
`